exercises/palindrome-products/PalindromeProductsExample.swift

```diff
@@ -0,0 +1,60 @@
+// Foundation not needed
+
+// Apple Swift version 2.1
+
+struct Palindromes {
+    
+    struct Palindrome {
```

Comment:
I take that back. You do need two different structs but we should conform one to the sequence type.